plugins/datadog_tracer/config/initializers/datadog_tracer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+if SamsonDatadogTracer.enabled?
+
+  SamsonDatadogTracer::IGNORED_URLS = Set[
+    '/ping',
+    '/streams',
+  ].freeze
+
+  require 'ddtrace'
+  Datadog.configure do |c|
+    # Tracer
+    c.tracer(
+      hostname:                ENV['STATSD_HOST'] || '127.0.0.1',
+      tags: {
+        env:                   ENV['RAILS_ENV'],
+        pod:                   ENV['ZENDESK_POD_ID'],
```

Comment:
don't think this is universal / don't think this matters ... can we leave it out ?